### ISSUE TITLE
Error when selecting 'No' for contributing

### ISSUE THREAD

**Issue by paulcpederson** [ngoldman/module-init#13]
When prompted:

```
? contributing: (Y/n)
```

If I enter `y` everything works great, but if I use `n` I get the following error:

```
[Error: missing required options: pkgContributing]
```

**Comment by ngoldman** [comment]
@paulcpederson thanks for catching that! will look into it

**Comment by ngoldman** [comment]
got rid of the error in d1e2ab1678bfcd52249c7cbb890e599e0c57cbd1 but it looks like the way we're doing the `templates` module makes it a little difficult to not actually create the `CONTRIBUTING.md`. Needs a little refactor.

**Comment by Flet** [comment]
Just hit this today myself.

**Comment by bcomnes** [comment]
Error... you must accept any and all contributions :stuck_out_tongue_winking_eye:

**Comment by ngoldman** [comment]
fixed in [1.0.0](https://github.com/ngoldman/module-init/releases/tag/v1.0.0)

Released as 1.0.0 because 0.x versions are sadly meaningless in semver and ignored by `^` & `~` -- see https://nodesource.com/blog/semver-tilde-and-caret#1-0-0-anxiety for more info.
